website-next/public/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/mstile-150x150.png"/>
+            <TileColor>#da532c</TileColor>
+        </tile>
+    </msapplication>
+</browserconfig>
```

Comment:
Oh wow, we're supporting IE now?! :sweat_smile:

Reply:
this is Windows 10 start menu support!!! malte's favorite OS

Reply:
Did you really just use my deviant preferences as an excuse for your procrastitask? That's low / genius.